runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankTagsPlugin.java

```diff
@@ -97,7 +97,7 @@ private void setTags(int itemId, String tags)
 		}
 		else
 		{
-			configManager.setConfiguration(CONFIG_GROUP, ITEM_KEY_PREFIX + itemId, tags);
+			configManager.setConfiguration(CONFIG_GROUP, ITEM_KEY_PREFIX + itemId, tags.toLowerCase());
```

Comment:
I dont think this wil fix the issue backwards, the toLowerCase should be called when comparing, not when saving